src/lib/components/profile/modal/Modal.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import cancelIcon from '$lib/icons/cancel-black.svg'
+  import { fade, scale } from 'svelte/transition'
+
+  export let isOpen: Boolean
+  export let title = ''
+  export let subtitle = ''
+  export let icon = ''
+
+  function closeModal() {
+    isOpen = false
+  }
+</script>
+
+{#if isOpen}
+  <div
```

Comment:
```suggestion
  <article
```
Me fumé un porro con esto